cylc/flow/network/resolvers.py

```diff
@@ -671,7 +674,7 @@ def force_spawn_children(
         Args:
             tasks: List of identifiers, see `task globs`
             outputs: List of outputs to spawn on
-            flow_num: Flow number to attribute the outputs.
+            flow_num: Flow number to attribute the outputs
```

Comment:
[Pernickity] IMO they should _all_ have full-stops. I've double checked and they do in the G**gl* style guide.
```suggestion
            tasks: List of identifiers, see `task globs`.
            outputs: List of outputs to spawn on.
            flow_num: Flow number to attribute the outputs.
```

Reply:
(Huh, your "suggestion" covers deleted lines too, so I can't commit it directly from GitHub UI)